webapp/src/app/user-data.service.ts

```diff
@@ -36,7 +35,7 @@ export class UserData extends Data {
     async load() {
         Log.lvl1("Loading data from", this.dataFileName);
         const values = await this.storage.getObject(this.dataFileName);
-        if (!values || values === {}) {
+        if (Object.keys(values).length === 0) {
```

Comment:
You can now check for `undefined`.
```suggestion
        if (values === undefined) {
```